synth.py

```diff
@@ -20,16 +20,39 @@
 
 logging.basicConfig(level=logging.DEBUG)
 
-# run the gapic generator
-gapic = gcp.GAPICGenerator()
+gapic = gcp.GAPICMicrogenerator()
 versions = ['v1beta1', 'v1', 'v1p2beta1']
+name = 'asset'
 for version in versions:
-  library = gapic.node_library(
-    'asset',
-    version,
-    config_path=f"artman_cloudasset_{version}.yaml",
-    artman_output_name=f"asset-{version}")
-  s.copy(library, excludes=['src/index.js', 'README.md', 'package.json'])
+    library = gapic.typescript_library(
+        name,
+        proto_path=f'google/cloud/{name}/{version}',
+        generator_args={
+            'grpc-service-config': f'google/cloud/{name}/{version}/cloud{name}_grpc_service_config.json',
+            'package-name': f'@google-cloud/{name}'
+        },
+        version=version)
+    # skip index, protos, package.json, and README.md
+    s.copy(
+        library,
+        excludes=['package.json', 'src/index.ts']
+    )
+
+versions = ['v1p1beta1']
```

Comment:
This is separate because there is no gRPC service config? It seems like a bug in protos. Let me figure this out.